commons/protocol/src/test/resource/saved-protocol/0.2/array/seq_int.json

```diff
@@ -0,0 +1 @@
+[1.0,2.0,3.0]
```

Comment:
this isn't desirable, we should be serializing ints without the ".0", though we should accept the ".0" on unpickle.